src/Tizen.NUI/src/internal/WebView/WebView.cs

```diff
@@ -662,6 +749,90 @@ public bool KeyEventsEnabled
             }
         }
 
+        /// <summary>
+        /// Background color of web page.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Vector4 DocumentBackgroundColor
+        {
+            get
+            {
+                return documentBackgroundColor;
+            }
+            set
+            {
+                documentBackgroundColor = value;
+                SetValue(DocumentBackgroundColorProperty, value);
```

Comment:
Why don't you put this line `documentBackgroundColor = value;` into the corresponding bindable property setter?
The logic should be same in both property setter and the bindable property setter.

Reply:
Done.